addons/subscription/models/subscription.py

```diff
@@ -0,0 +1,123 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+# TODO:
+#   Error treatment: exception, request, ... -> send request to user_id
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+
+def _get_document_types(self):
+    self.env.cr.execute('select m.model, s.name from subscription_document s, ir_model m WHERE s.model_id = m.id order by s.name')
+    return self.env.cr.fetchall()
```

Comment:
Simple search method should do the job. Something like :
```
def _get_document_types(self):
    return [(doc.model_id.model, doc.name) for doc in self.env['subscription.document'].search([], order='name')]
```
Can you check the reason for using query here, and replace it with search if it is not really necessary ?